docs/aliases/awful_client.lua

```diff
@@ -1,6 +1,6 @@
 ---------------------------------------------------------------------------
---- This module is deprecated, use `client`
---  ===============================
+--- This module documentation has been merged with the `client` module
+--  ==================================================================
```

Comment:
Please add a dot at the end of the sentence (also below).

And I think it might render better without the underlining?!